### PR TITLE
Sustained Immunity Update

### PR DESCRIPTION
**Pull request by RLordan**
One or two new papers added to the sustained immunity section.

### Suggested reviewers (optional)

@rando2 feel free to suggest others

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/10.diagnostics.md
Comment (agitter) [inline content/10.diagnostics.md:238]:
```suggestion
Gaebler et al. have shown that memory B cells specific to the RBD remain unaltered and exhibit clonal turnover and antibody sequence evolution 6 months post infection indicative of prolonged germinal cell reactions  [@doi:10.1038/s41586-021-03207-w]. 
```

File: content/10.diagnostics.md
Comment (agitter) [inline content/10.diagnostics.md:232]:
```suggestion
These findings are in accordance with other studies that show that the majority of seroconverters have detectable, albeit decreasing levels of neutralizing antibodies at least 3-6 months post infection [@doi:10.1126/science.abd7728; @doi:10.1038/s41564-020-00813-8; @doi:10.1038/s41467-021-21444-5].
```

**Review by agitter — CHANGES_REQUESTED** [review]
I can't give a scientific review, but the text reads well.

File: content/10.diagnostics.md
Comment (RLordan) [reply content/10.diagnostics.md:232]: Oops, thanks

**Comment by RLordan** [comment]
> I can't give a scientific review, but the text reads well.

Thanks @agitter !

File: content/10.diagnostics.md
Comment (rando2) [inline content/10.diagnostics.md:240]:
```suggestion
Indeed, Wheatley et al. showed that S-specific IgG^+^ memory B cells consistently increase over time and by 4 months comprise approximately 0.8% of all IgG^+^ memory B cells, which may indicate cellular immune memory to even mild-to-moderate COVID-19 infection [@doi:10.1038/s41467-021-21444-5]. 
```

File: content/10.diagnostics.md
Comment (rando2) [inline content/10.diagnostics.md:246]:
```suggestion
Indeed, the abundance of S-specific memory CD4^+^ T cells over time was similar to that of SARS-CoV-2-specific CD4^+^ T cells overall [@doi:10.1126/science.abf4063].
```

**Review by rando2 — COMMENTED** [review]
Hey @RLordan, apologies it took so long for me to put together a scientific review! This is very much not my area but I tried to make sure I got the same understanding from the original paper and your summary where possible, so I had a few questions. You definitely know more about this than I do, though!

File: content/10.diagnostics.md
Comment (RLordan) [reply content/10.diagnostics.md:217]: Generally speaking, the consensus is that yes infection does confer sustained immunity but that after the 6-8 month period the immunity does wane and there is increased risk of reinfection for some. It is not clear though yet, whether there is still protection from reinfection beyond that time point, and we really don't know what the story will be with the t-cell and b-cells pool. My guess is there must be some protection, but how protective that is to prevent reinfection, I really do not know. The group that produced the Gaebler study have kept some patients to do a 1 and 2 year post infection study, which will certainly provide answers to that. 

I hear your concerns though, inferring protection from these types of studies is not always indicative of reality, but my perception is that examining antibody responses and assessing neutralizing activity seems to be the gold standard for these folks.  

PS I highly recommend you listen to Christian Gaebler who is on an episode of the podcast TWIV, which I also recommend. He gives an overview of his study and what is to come for sustained immunity research with SARS-CoV-2

File: content/10.diagnostics.md
Comment (RLordan) [reply content/10.diagnostics.md:226]: I got that they looked at antibodies up to 6 months, and reported a reduction of antibodies form 6-8 months. They reported the T- and B-cells from 6 to 8 months. 

They adapt that to conclude that immunity will last 6-8 months. Do we need to rephrase here if it is not clear?

File: content/10.diagnostics.md
Comment (RLordan) [reply content/10.diagnostics.md:227]: Haha nitpicking welcome. 

The line I wrote refers to page 284 on the fourth line, referring to samples obtained from 15 patients for the two month period post 6 month collection (6-8 months). They also suggest that they still had neutralizing activity (page 286/figure S1).

Have a look and see if you think. :)

File: content/10.diagnostics.md
Comment (RLordan) [reply content/10.diagnostics.md:228]: So one possibility is that people didn't seroconvert, then they had no rise in antibodies in the first place. However, they only included people that had seroconverted. 13 people were excluded from this study who did not seroconvert. You only see that in the supplementary submission files. 

I see what you are saying though. It is not possible to tell from figure 1 whether one or two participants did not follow that trend.

File: content/10.diagnostics.md
Comment (RLordan) [reply content/10.diagnostics.md:238]: Germinal center/centre, I also have no clue what the right spelling is haha I went with the former- they are temporary structures in the lymph nodes following immunization where B cells do their business.  
```suggestion
Gaebler et al. have shown that memory B cells specific to the RBD remain unaltered and exhibit clonal turnover and antibody sequence evolution 6 months post infection indicative of prolonged germinal center reactions [@doi:10.1038/s41586-021-03207-w]. 
```

File: content/10.diagnostics.md
Comment (RLordan) [reply content/10.diagnostics.md:230]: I assumed it might have been earlier when discussing antibody diagnostics. Only way to confirm whether you had infection prior to receiving a vaccine. Because the vaccine induces antibodies for spike. Super cool

File: content/10.diagnostics.md
Comment (RLordan) [reply content/10.diagnostics.md:235]: lifespan, longevity? Although I don't think the lifespan has been fully assessed as memory B-cells can sometimes last for decades, but the abundance has been assessed a lot...

**Comment by RLordan** [comment]
Hey @rando2, a few responses for you. If you get a moment, double check if my replies make sense to you. It is a difficult topic :/

File: content/10.diagnostics.md
Comment (rando2) [reply content/10.diagnostics.md:217]: Thank you so much for explaining @RLordan! And thank you for the rec, I just subscribed to that podcast!

That definitely makes sense! Maybe something we could mention in the conclusions for this paper in terms of future directions? That's fantastic they're geared up for a longitudinal study!

File: content/10.diagnostics.md
Comment (rando2) [reply content/10.diagnostics.md:226]: That makes sense! That helps me understand why they portrayed it as 6-8!

File: content/10.diagnostics.md
Comment (rando2) [reply content/10.diagnostics.md:227]: Thank you so much @RLordan! I see your original numbers!

**Review by rando2 — APPROVED** [review]

File: content/10.diagnostics.md
Comment (rando2) [reply content/10.diagnostics.md:235]: I'm definitely fine leaving it the way you wrote it, it sounds like additional studies are needed to talk about the timecourse!

**Comment by rando2** [comment]
@agitter I just wanted to check if you have any additional concerns with this PR -- I checked the html file and it looks like there are no broken references

**Comment by RLordan** [comment]
For some reason I couldn't reply directly to your comment @rando2 

Sure I think mentioning that longitudinal studies are going on will be useful. I'm sure we can reference clinicaltrials.gov

PS Susan Weiss was on TWIV this week!

**Comment by rando2** [comment]
Thanks @RLordan! I added a note to the conclusions! I'll definitely check it out, my favorite podcast just went on hiatus so it's perfect timing! 

I am happy to see this merged in but for GitHub reasons just waiting to confirm that Tony is okay with me doing that!

**Review by agitter — APPROVED** [review]
> I am happy to see this merged in but for GitHub reasons just waiting to confirm that Tony is okay with me doing that!

Yes, I'm okay merging.  I'll double-check all references and proofread end-to-end right before submission, so don't let me hold up merging these sections.

**Pull request merged**
